src/stactools/sentinel2/constants.py

```diff
@@ -80,6 +80,11 @@
                 description='Band 9 - Water vapor',
                 center_wavelength=0.945,
                 full_width_half_max=0.026),
+    'B10':
+    Band.create(name='B10',
+                description='Band 10 - SWIR - Cirrus',
+                center_wavelength=1.3735,
+                full_width_half_max=0.026),  # TODO: find out fwhm
```

Comment:
The full_width_half_max for Band 10 is 0.075 for Sentinel-2A (the wavelength numbers are slightly different for Sentinel-2B!).

Reply:
Thanks @jonas-eberle! Worked this in.